src/ipynb/spark_ml.ipynb

```diff
@@ -34,7 +34,7 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # Delete the session teamplate
</pre><p>typo: teamplate -&gt; template</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/terraform-google-analytics-lakehouse/pull/116/discussion/#comment-1471499169'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>